vizro-core/docs/pages/user-guides/actions.md

```diff
@@ -170,7 +173,7 @@ Here is an example of how to configure a chart interaction when the source is a
         ```
     === "app.yaml"
         ```yaml
-        # Still requires a .py to add data to the data manager and parse YAML configuration
+        # Still requires a .py to register data connector in Data Manager and parse yaml configuration
```

Comment:
I don't remember what Antony said again, but as far as I remember we shouldn't use data connector, data set etc. but just data? Could you double-check in his PR on the data manager 🤔

Reply:
Thanks, this was an issue in the merge which I thought I'd fixed. I've grepped all the source for data connector/dataset and done a line by line comparison of Antony's changes and mine, and I think/hope/pray we are all good now 🙏